docs/setup/embedding.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[embedding]]
+== Embedding {kib}
+
+Once you create a dashboard, or a visualization you surely might want to share it with your colleagues or friends at some point. The easiest way to do that is to share a direct link to your dashboard or visualization, but not everyone might have access to your {kib} or even know what {kib} is. Moreover, sometimes you might want to give more context to the thing you're sharing, for example, attach analysis, or images, or links, and {kib} isn't always the best tool for this type of job.
+
+If you have an internal company website or just a personal webpage where it'd make more sense to display the content you created in {kib}, then try out the {kib} embedding functionality. You can embed a {kib} dashboard or visualization using an HTML code snippet generated with the *Copy iFrame code* button in the *Share > Embed code* menu in *Dashboard* or *Visualize*.
+
+NOTE: Embedding of any other part of {kib} is also generally possible, but you may need to craft the proper HTML code manually.
+
+image::images/embed-kibana.png[Generate an HTML snippet to embed {kib}, align=center]
+
+[float]
+[[embedding-security]]
+=== Configure security
+
+Embedding content through iframes requires careful consideration to minimize security risks. By default, modern web browsers enforce the
+https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy[same-origin policy] to restrict the behavior of framed pages. When
+{stack-security-features} are enabled on your cluster, you must relax this constraint for cookies as described below for {kib} to function
```

Comment:
We avoid the word "below" for accessibility reasons.  Can you link to the section?